tapeagents/utils.py

```diff
@@ -115,6 +118,30 @@ def encode_image(image_path):
         return base64.b64encode(image_file.read()).decode("utf-8")
 
 
+def get_relative_path_to_root(path, root_path):
+    """
+    Example:
+    path=/a/b/c/d/file.txt
+    root_path=b/c/
+    returns d/file.txt
+    """
+    # Normalize the paths
+    path = os.path.normpath(path)
+    root_path = os.path.normpath(root_path)
+
+    # Find the index of root_path in path
```

Comment:
I think you can just use `os.path.relpath()` here. It converts absolute path from the first argument to the relative path relatively to the second argument